handbook/queries/get-installed-chrome-extensions

```diff
@@ -0,0 +1,11 @@
+# Get installed Chrome Extensions
+
+List installed Chrome Extensions for all users
+
+### Support
+MacOS
```

Comment:
```suggestion
macOS
```

Reply:
Very small nit. Lowercase "m" on "macOS." Looks great to me after this small fix!